src/assets/wise5/authoringTool/node/editRubric/edit-rubric.component.ts

```diff
@@ -1,50 +1,42 @@
+import { Component, OnInit } from '@angular/core';
 import { insertWiseLinks, replaceWiseLinks } from '../../../common/wise-link/wise-link';
 import { ConfigService } from '../../../services/configService';
 import { TeacherDataService } from '../../../services/teacherDataService';
 import { TeacherProjectService } from '../../../services/teacherProjectService';
+import { UpgradeModule } from '@angular/upgrade/static';
 
-class EditRubricComponentController {
+@Component({
+  selector: 'edit-rubric',
+  templateUrl: 'edit-rubric.component.html'
+})
+export class EditRubricComponent implements OnInit {
   node: any;
   nodeId: string;
   rubric: string;
 
-  static $inject = [
-    '$state',
-    'ConfigService',
-    'ProjectService',
-    'TeacherDataService',
-    'UtilService'
-  ];
-
   constructor(
-    private $state: any,
-    private ConfigService: ConfigService,
-    private TeacherProjectService: TeacherProjectService,
-    private TeacherDataService: TeacherDataService
+    private configService: ConfigService,
+    private teacherProjectService: TeacherProjectService,
+    private teacherDataService: TeacherDataService,
```

Comment:
I think we can shorten the variable names here, since there aren't any other ProjectService or DataService in this class.
```suggestion
    private projectService: TeacherProjectService,
    private dataService: TeacherDataService,
```